native/dlasq3.go

```diff
@@ -0,0 +1,153 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package native
+
+import "math"
+
+// Dlasq3 checks for deflation, computes a shift (tau) and calls dqds.
+// In case of failure it changes shifts, and tries again until output
+// is positive.
+func (impl Implementation) Dlasq3(i0, n0 int, z []float64, pp int, dmin, sigma, desig, qmax float64, nFail, iter, nDiv int, ttype int, dmin1, dmin2, dn, dn1, dn2, g, tau float64) (
+	i0Out, n0Out, ppOut int, dminOut, sigmaOut, desigOut, qmaxOut float64, nFailOut, iterOut, nDivOut, ttypeOut int, dmin1Out, dmin2Out, dnOut, dn1Out, dn2Out, gOut, tauOut float64) {
+	// Note: The documentation says dmin is an output only, but if that is so it is
```

Comment:
I still see it as such http://www.netlib.org/lapack/explore-html/db/d5b/dlasq3_8f_source.html but I also submitted a bug that was supposedly fixed http://icl.cs.utk.edu/lapack-forum/viewtopic.php?f=13&t=4830 so probably depends on where one looks.